### PR TITLE
real-time updates guide

### PR DESCRIPTION
**Pull request by CLAGOR**

### CODE REVIEW

File: docs/Feature_guides/Real-time_updates.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+### Back-end
+
+##### Define the subscription class
+
+```ruby
+# plugins/easyproject/easy_plugins/easy_agile/api/easy_graphql/subscriptions/easy_product_backlog_board_move_pbi.rb
+module EasyGraphql
+  module Subscriptions
+    class EasyProductBacklogBoardMovePbi < ::EasyGraphql::Subscriptions::EasyProductBacklogBoard
```
Comment (jiribruk) [inline docs/Feature_guides/Real-time_updates.md:115]: missing code from ::EasyGraphql::Subscriptions::EasyProductBacklogBoard .  

create an example that does not exist here, which will contain the code from the original subscription

```suggestion
    class EasyProductBacklogBoardMovePbi < ::EasyGraphql::Subscriptions::Base
    
   description 'Subscription for receiving updates about PBI position/status in EasyProductBacklogBoard'

      subscription_scope :easy_sprint_board_id
      argument :easy_sprint_board_id, ID, required: true
      
       field :result, Subscriptions::Results::MovePbiResult, null: true

      def subscribe(easy_sprint_board_id:)
        {
          easy_sprint_board: find_board(easy_sprint_board_id)
        }
      end

      private

      def find_board(id)
        ::EasySprintBoard.find_by(id: id)
      end
```

File: docs/Feature_guides/Real-time_updates.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+
+### Back-end
+
+- `gem 'graphql'`
+- working websockets (one is enough) -> GraphQL subscriptions use [ActionCable](https://guides.rubyonrails.org/action_cable_overview.html) internally so `cable.yml` needs to be configured
+
+## Implementation
+
+_PATTERN:_ Use multiple context-oriented subscriptions instead of a single big one. They're easier to manage and customize avoiding `if` conditions.\
+In our backlog board example we use 5 different subscriptions:
```
Comment (jiribruk) [inline docs/Feature_guides/Real-time_updates.md:30]: So that subscriptions for individual boards do not conflict. We use subscription_scope.

**Comment by luk4s** [comment]
@CLAGOR make sure that there is a link somewhere (I recommend use menu) which pointing to this file 😉

File: docs/Feature_guides/Real-time_updates.md (hunk 1 above)
Comment (CLAGOR) [reply docs/Feature_guides/Real-time_updates.md]: Rewritten to include more details.

File: docs/Feature_guides/Real-time_updates.md (hunk 2 above)
Comment (CLAGOR) [reply docs/Feature_guides/Real-time_updates.md:30]: After testing, subscription_scope will be removed so we leave it out.

**Comment by CLAGOR** [comment]
> @CLAGOR make sure that there is a link somewhere (I recommend use menu) which pointing to this file 😉

done, thanks :)

**Review by luk4s — APPROVED** [review]

**Pull request merged**